docs/admin/install.rst

```diff
@@ -44,6 +45,34 @@ A basic knowledge of the Qubes OS is helpful.
 Pre-install tasks
 -----------------
 
+Verify the SecureDrop server configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In order to be used with SecureDrop Workstation, your instance must be running the latest version of SecureDrop, and the server configuration must have been updated to allow for HTTP ``DELETE`` requests. The configuration change to enable this was added in the ``0.13.0`` version of SecureDrop, released on May 29 2019. If your instance was created using this or a later version, it has the necessary changes. If not, then the ``./securedrop-admin install`` command must have been run from an Admin Workstation updated with the ``0.13.0`` code or later. To check this:
+
+- Use an Admin Workstation USB to boot into Tails, with the persistent volume unlocked and an administration password set.
+- Navigate to **Applications ▸ System Tools ▸ Terminal** to open a terminal.
+- Verify that the *Journalist Interface* Apache configuration allows for HTTP ``DELETE`` using the following command:
+
+  .. code-block:: sh
+
+    ssh app "sudo cat /etc/apache2/sites-enabled/journalist.conf | grep '^  <LimitExcept GET POST HEAD DELETE>$'"
+
+- If your instance is configured correctly, this command will output two lines as follows:
+
+  .. code-block:: none
+
+     <LimitExcept GET POST HEAD DELETE>
+     <LimitExcept GET POST HEAD DELETE>
+
+- If not, then you will need to:
+
+  - Update the Admin Workstation to the current SecureDrop release version.
```

Comment:
Maybe add something like "by following the applicable upgrade guide in our documentation"?

Reply:
:heavy_check_mark: